src/python/TaskWorker/Actions/PostJob.py

```diff
@@ -519,22 +519,25 @@ def inject_to_aso(self):
 
 
         # TODO: Add a method to resolve a single PFN.
+        import pickle
+        with open('taskinformation.pkl', 'rb') as fd:
+            task = pickle.load(fd)
```

Comment:
Not a fan of this, but I think it does the job for the moment. the point is that `taskinformation.pkl` is only used by the automatic splitting and it is a new dependency for the PJ. I will merge this and open a github issue about finding a better way to do it.